support-data-rest/src/main/java/org/uniprot/api/support/data/configure/controller/UniParcConfigureController.java

```diff
@@ -97,4 +97,9 @@ public UniParcConfigureController(UniParcConfigureService service) {
     public List<UniParcDatabaseDetail> getUniParcDatabaseDetails() {
         return service.getAllUniParcDatabaseDetails();
     }
+
+    @GetMapping("/database-result-fields")
+    public List<UniProtReturnField> getDatabaseResultFields() {
```

Comment:
should we change this path to crossReference instead of database? also, should we change the path?

Reply:
changed it to /entry-result-fields